pages/linux/lsblk.md

```diff
@@ -0,0 +1,27 @@
+# lsblk
+
+> lsblk lists information about all available or the specified storage devices.
```

Comment:
Could we shorten this into "lists information about storage devices"? That sounds snappier while still in essence containing the above.